paper/paper.md

```diff
@@ -31,17 +31,40 @@ bibliography: paper.bib
 ---
 
 # Summary
-Analyzing and identifying genomic variants is particularly helpful in case of linking the disease onset to the genomic predictions. Genomic variants play a key role in the identification and characterization of the disease onset and also allow for interlinking genomic variability and hypothesis testing. eVai, which is a variant annotation platform provides annotation of the genomic variants using several backend open source databases and tools. eVaiutilities proposed in this paper, is a post-eVAI analyzer for population scale analysis for eVAI variant annotation. It allows the analysis of the genomic variants further such as analyzing the multiple genomic annotated variants, reference and alternate allele, enabling coordinate search, coordinate search with specified variants and annotation search across a large number of population. The availability of the command line parameters allows for a large scale analysis across the several population files. eVAI has been used by many people and has been very well cited, so the lack of the use of the command line for the high-throughput analysis promoted to develop eVaiutilities. Email: gsablok@ibch.poznan.pl
+Analyzing and identifying genomic variants is particularly helpful when linking the _disease onset_ to the genomic predictions. Genomic variants play a key role in the identification and 
+characterization of the _disease onset_ and also allow for interlinking genomic variability and hypothesis testing. eVai, a variant annotation platform, provides annotation of the genomic variants 
+using several backend open source databases and tools. eVaiutilities proposed in this paper, is a ~~post-eVAI~~ analyzer for population scale analysis for eVAI variant annotation. It allows the 
+_analysis of the genomic variants further such as analyzing the multiple genomic annotated variants_, reference and alternate allele, enabling coordinate search, coordinate search with specified 
+variants and annotation search across _a large number of population_. The availability of the command line parameters allows for a large scale analysis across the several population files. _eVAI 
```

Comment:
This is not correct grammatically: "a large number of population" - did you mean many different populations?